dvc/api.py

```diff
@@ -214,6 +215,267 @@ def read(path, repo=None, rev=None, remote=None, mode="r", encoding=None):
         return fd.read()
 
 
+def params_show(
```

Comment:
@daavoo Let's please at least put it into a separate file, because it is pretty giant. E.g. `dvc/api/params.py`, keeping the imports the same.